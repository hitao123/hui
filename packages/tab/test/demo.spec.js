import { renderToString } from '@vue/server-test-utils';
import TabDemo from '../demo/index.vue';

describe('Dialog component', () => {
  it('should show Button Demo', () => {
    expect(renderToString(TabDemo)).toMatchSnapshot();
  });
});
