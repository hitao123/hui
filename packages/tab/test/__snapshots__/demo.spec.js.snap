// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="tab-demo">
  <div>
    <div class="h-hairline--top-bottom">
      <div class="h-tabs__nav">
        <div class="h-tabs__line"></div>
      </div>
    </div>
    <div class="h-tabs__content">
      <div class="h-tab__pane" style="display:none;">
        <!---->
        <!---->
      </div>
      <div class="h-tab__pane" style="display:none;">
        <!---->
        <!---->
      </div>
      <div class="h-tab__pane" style="display:none;">
        <!---->
        <!---->
      </div>
      <div class="h-tab__pane" style="display:none;">
        <!---->
        <!---->
      </div>
    </div>
  </div>
</div>
`;
